### PR TITLE
@alloy - [EOY] Analytics

### PR DESCRIPTION
**Pull request by broskoski**
Closes https://github.com/artsy/force/issues/568
Follow button already has analytics on it, this adds the other two events.

screenshot 2016-12-20 15 29 43

### CODE REVIEW

**Comment by alloy** [comment]
The artist click event doesn’t include the `id`, only the `slug`, the same two properties are also required for the follow event. Is there a reason to only include one of them?

**Comment by broskoski** [comment]
@alloy we actually don't have the artist `_id` only the slug: https://www.artsy.net/eoy_2016/data

**Comment by alloy** [comment]
Ah, I see.

@anipetrov Is either slug/id good or do you really prefer both?

File: analytics/eoy_2016.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+(function () {
+  'use strict'
+
+  // DOM events
+  var $document = $(document)
+
+  $document.on('click', '.eoy-artist-feature .avant-garde-button-white', function (e) {
+    analytics.track('Clicked EOY promotion read article', {
+      destination_path: $(this).attr('href'),
```
Comment (alloy) [inline analytics/eoy_2016.js]: Just to be sure, does this not need to be purely the path component of the URL?

Comment (broskoski) [inline analytics/eoy_2016.js]: Yeah, not sure. I'm taking directly whats in the `article_link` field in  https://www.artsy.net/eoy_2016/data

Comment (alloy) [inline analytics/eoy_2016.js]: Gotcha.

@anipetrov Got any opinion on this one as well?

**Comment by anipetrov** [comment]
@alloy would prefer both if not too much trouble! id is great for connecting to other tables, slug is good for quickly scanning the data for particular entities

**Comment by broskoski** [comment]
@alloy looks like we will have to pull `_id` for the artists and enter them in manually. Any good tribute candidates to do that :)

**Comment by anipetrov** [comment]
oh if it's a lot of work we can leave out the slug for this particular case!

**Comment by alloy** [comment]
@broskoski I can easily do that from the terminal, is there an easy way for me to just upload a new version of the JSON outside of manually editing in the JSONPage UI?

**Comment by broskoski** [comment]
Thanks for the terminal lifehack @alloy 👍

**Comment by alloy** [comment]
Alrighty then 👌

**Pull request merged**